test/docker/node20/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM rishabhpoddar/supertokens_node_driver_testing_node_20
```

Comment:
Assumed the image will be created with the name `supertokens_node_driver_testing_node_201